app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -199,8 +200,26 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
     submit,
     values,
   } = props
+
   const { arbitrator, category, collateral, funding, loadedQuestionId, outcomes, question, resolution, spread } = values
 
+  const { markets } = useGraphMarketsFromQuestion(loadedQuestionId || '')
+
+  const [currentToken, setCurrentToken] = useState({ exists: false, token: '', marketAddress: '' })
+  useEffect(() => {
+    const rinkebyDai = '0x5592ec0cfb4dbc12d3ab100b257153436a1f0fea'
+    const mainnetDai = '0x6b175474e89094c44da98b954eedeac495271d0f'
```

Comment:
Let's move these to the constants file

Reply:
I would actually name these as `DEFAULT_TOKEN_ADDRESS` and `DEFAULT_TOKEN_ADDRESS_RINKEBY` so that if the default token is ever changed, we would know to update these values and this wouldn't break